features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -904,7 +904,6 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             }
             if (isFeatureEnabled(featureConfig?.applications,
                 ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_SHARED_ACCESS"))
-                 && application?.templateId != ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
```

Comment:
Do we have any idea why this was disabled previously?

Reply:
This was disabled earlier since it was not supported. However i have tested the flows and are working as expected when shared in IS 7.1
cc: @NipuniBhagya @DilshanSenarath

Reply:
Ack